code/modules/mob/living/simple_animal/friendly/hamster.dm

```diff
@@ -45,7 +45,7 @@
 		vector_disease = new disease
 		message_admins("Vector was roundstart infected with [vector_disease.name]. Don't lynch the virologist!")
 		log_game("Vector was roundstart infected with [vector_disease.name].")
-	var/datum/disease/advance/R = new /datum/disease/advance/random(rand(1, 3))
+	var/datum/disease/advance/R = new /datum/disease/advance/random(rand(2, 5), rand(3,9))
```

Comment:
ditto. if no symptom level number is given it caps at 9 by default

Reply:
Altered Disease Generation to have a minimum level. Changing disease generation for maints to match.